mozilla/blurts-server
# only use Nimbus in stage and prod APP_ENV

# Description

Stop trying to use Nimbus/Cirrus non-stage/prod environments. Code trying to fetch the list of features will get `Promise<undefined>` back instead.

# How to test

Run local development and observe the console is not throwing exceptions on every page load about not being able to connect to Cirrus sidecar URL.

# Checklist (Definition of Done)
